central/sac/authorizer/builtin_scoped_authorizer.go

```diff
@@ -125,9 +125,13 @@ func (a *accessModeLevelScopeCheckerCore) SubScopeChecker(scopeKey sac.ScopeKey)
 	if !ok {
 		return errorScopeChecker(a, scopeKey)
 	}
-	resource, ok := resources.MetadataForResource(permissions.Resource(scope.String()))
+	res := permissions.Resource(scope.String())
+	resource, ok := resources.MetadataForResource(res)
 	if !ok {
-		return sac.ErrorAccessScopeCheckerCore(errors.Wrapf(ErrUnknownResource, "on scope key %q", scopeKey))
+		resource, ok = resources.MetadataForInternalResource(res)
+		if !ok {
+			return sac.ErrorAccessScopeCheckerCore(errors.Wrapf(ErrUnknownResource, "on scope key %q", scopeKey))
+		}
```

Comment:
```suggestion
	}
	if !ok {
		return sac.ErrorAccessScopeCheckerCore(errors.Wrapf(ErrUnknownResource, "on scope key %q", scopeKey))
```
reduces the indentation level